WireAnalytics/Tests/WireAnalyticsTests/AnalyticsManagerTests.swift

```diff
@@ -77,11 +77,12 @@ class AnalyticsManagerTests: XCTestCase {
 
         // THEN
         XCTAssertEqual(analyticsService.endSession_Invocations.count, 1)
-        guard analyticsService.changeDeviceID_Invocations.count == 1 else {
-            XCTFail("Expected 1 invocation of changeDeviceID, but got \(analyticsService.changeDeviceID_Invocations.count)")
+        guard analyticsService.changeDeviceIDMergeData_Invocations.count == 1 else {
+            XCTFail("Expected 1 invocation of changeDeviceIDMergeData, but got \(analyticsService.changeDeviceIDMergeData_Invocations.count)")
             return
         }
-        XCTAssertEqual(analyticsService.changeDeviceID_Invocations[0], "testUser123")
+        XCTAssertEqual(analyticsService.changeDeviceIDMergeData_Invocations[0].id, "testUser123")
+        XCTAssertFalse(analyticsService.changeDeviceIDMergeData_Invocations[0].mergeData)
```

Comment:
I think we also need a test case where we assert this is true. Could be in a future PR.

Reply:
Yep!